libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker-input.directive.ts

```diff
@@ -71,16 +69,17 @@ export class SkyColorpickerInputDirective
    * @deprecated
    */
   @Input()
-  public set initialColor(value: string) {
+  public set initialColor(value: string | undefined) {
     /* istanbul ignore else */
-    if (!this._initialColor && !this.modelValue) {
+    if (!this.#_initialColor && !this.#modelValue) {
       this.writeValue(value);
     }
-    this._initialColor = value;
+
+    this.#_initialColor = value;
   }
 
   public get initialColor(): string {
-    return this._initialColor || SKY_COLORPICKER_DEFAULT_COLOR;
+    return this.#_initialColor || SKY_COLORPICKER_DEFAULT_COLOR;
```

Comment:
Possible to rework the getter so it doesn't return default values?

Reply:
This was one place where I couldn't do that because of the convoluted logic around initial color. I tried adding a comment to the PR with that info but it's not appearing in this view for some reason.